content/practices/event-storming-process-modelling/index.md

```diff
@@ -8,28 +8,27 @@ description: "A workshop for facilitating discussions around an outcome based vi
 # Note: remove any tags that are not relevant.
 tags: ["Scoping", "Kickoff", "Discovery", "Framing", "Inception", "Transition", "Modernization", "Delivery"]
 length: "1-2 hours"
-participants: "business: focused group of business analysts or executives or domain experts; core team: engineering, product management, design"
+participants: "Core Team and a focused group of Business Analysts, Executives or Domain Experts"
 # custom "cover" image example: "boris/boris.png"
 image: "default-cover.png" 
 lastmod: "2021-05-21"
 why: 
-- To break down and model business processes, aligning balanced teams with stakeholders and SMEs on business practice.
+- To capture and model business processes, aligning balanced teams with stakeholders and SMEs on business practice.
 - To identify user interaction within a process and identify where user journeys are required.
-- To align with a common and understandable language.
+- To align balanced teams with stakeholders and SMEs around a common and understandable language.
 - To build a feature oriented context for future development.
-- To draft, communicate and validate lean product assumption, of how the process should behave, with stakeholders and SMEs.
+- To draft, communicate and validate lean product assumptions, with stakeholders and SMEs.
 - To identify a thin slice and first iteration with prioritize lean assumptions.
 - To create an algorithmic model of a business processes.
```

Comment:
I'd say _process_ instead of _processes_.

Reply:
good catch thanks